service/src/test/java/com/ge/predix/controller/test/PolicyEvalWithGraphDbControllerIT.java

```diff
@@ -64,10 +68,20 @@
     private static final String POLICY_SET_URL = "v1/policy-set";
     private static final String SUBJECT_URL = "v1/subject";
     private static final String RESOURCE_URL = "v1/resource";
+    private static long TRAVERSAL_LIMIT;
```

Comment:
this is not a constant, just use a method variable in the only test method its used.